scripts/send_to_helix.py

```diff
@@ -44,6 +44,7 @@ class PerfSendToHelixArgs:
     wait_for_work_item_completion: bool = True
     creator: str = ""
     helix_results_destination_dir : Optional[str] = None
+    fail_on_test_failure: bool = False
```

Comment:
fail_on_test_failure does not appear to be used/set in run_performance_job. Double checking that this is on purpose.

Reply:
I'm now setting the value properly, thanks for catching it. The value is used by the Helix SDK and is used when doing Compare runs.